report/3.Theory/ObjectRecognition.tex

```diff
@@ -137,13 +137,97 @@ \subsection{Tensorflow}
 
 Tensors can also be tuples of different shapes, and when applying levels of abstraction, which is what Keras does, it is possible to use it for image recognition and other more complex problems.
 
-
 \subsection{OpenCV}
 OpenCV(Open Source Computer Vision Library), is a free-to-use library for C++, Python, and Java, which can be used for a lot of common computer vision tasks out of the box\cite{opencv}.
```

Comment:
change from "a lot of " to "a range of common computer.."

Reply:
Dafuq? Har du selv rettet?
Also why link til buildartifact?